docs/running-on-kubernetes.md

```diff
@@ -1356,6 +1356,15 @@ See the [configuration page](configuration.html) for information on Spark config
   </td>
   <td>3.3.0</td>
 </tr>
+<tr>
+  <td><code>spark.kubernetes.job.queue</code></td>
+  <td><code>(none)</code></td>
```

Comment:
Shall we follow the other `(none)`s in this document? It seems that we don't codify `(none)` here.